src/main/java/com/neprozorro/repository/LotItemInfoRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.neprozorro.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.UUID;
+
+public interface LotItemInfoRepository extends JpaRepository<com.neprozorro.model.LotItemInfo, UUID> {
```

Comment:
Why full path for LotItemInfo? If we have more than one classes like this we should rename it

Reply:
Done